po/pl.po

```diff
@@ -9195,62 +9195,62 @@ msgstr "%s%s[%s%s%s] %s%s%s uciszony"
 
 #, c-format
 msgid "%s%s: command \"%s\" not found:"
-msgstr "%s%s: nie znaleziono komendy \"%s\":"
+msgstr "%s%s: nie znaleziono komendy „%s\”:"
```

Comment:
There's a trailing `\` here (detected by automatic tests on Travis: https://travis-ci.org/weechat/weechat/jobs/187174935#L1050).
Could you please fix that please?
Thanks.